gradle-plugins/biz.aQute.bnd.gradle/src/main/java/aQute/bnd/gradle/BndPlugin.java

```diff
@@ -211,18 +211,10 @@ public void apply(Project project) {
 						.dirs(bndProject.getGenerate()
 							.getOutputDirs())
 						.withPropertyName("generateOutputs");
-					t.doLast("generate", new Action<>() {
-						@Override
-						public void execute(Task tt) {
-							try {
-								bndProject.getGenerate()
-									.generate(false);
-							} catch (Exception e) {
-								throw new GradleException(String.format("Project %s failed to generate", bndProject.getName()), e);
-							}
-							checkErrors(tt.getLogger());
-						}
-					});
+					t.doLast("generate", task(tt -> {
```

Comment:
You cannot use lambdas for gradle actions. This breaks gradle since it needs to see anon inner classes for it internal caching. 

https://github.com/gradle/gradle/issues/10751